src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderLabel.cs

```diff
@@ -3,33 +3,55 @@
 
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Runtime.Versioning;
 
 namespace System.Security.Cryptography.Cose
 {
+    /// <summary>
+    /// Represents a COSE header label.
+    /// </summary>
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct CoseHeaderLabel : IEquatable<CoseHeaderLabel>
     {
         internal string LabelName => LabelAsString != null ? $"\"{LabelAsString}\"" : LabelAsInt32.ToString();
         private string DebuggerDisplay => $"Label = {LabelName}, Type = {(LabelAsString != null ? typeof(string) : typeof(int))}";
 
         // https://www.iana.org/assignments/cose/cose.xhtml#header-parameters
+        /// <summary>
+        /// Gets a <see cref="CoseHeaderLabel"/> that represents the known header parameter "alg".
+        /// </summary>
         public static CoseHeaderLabel Algorithm => new CoseHeaderLabel(KnownHeaders.Alg);
```

Comment:
I feel like I already saw that @vcsjones pointed this out, but it's not on screen here, so I'll say it(, too).

Properties also need `<value>` documentation, which per https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value is (for simple properties) just repeating the summary without the "Gets [or sets]" (though sometimes that requires a bit of massaging)

```suggestion
        <value>A header label that represents the known header parameter "alg".</value>
        public static CoseHeaderLabel Algorithm => new CoseHeaderLabel(KnownHeaders.Alg);
```